code-challenges.js

```diff
@@ -57,12 +95,30 @@ const givenValue1 = 42
 const givenValue2 = 10
 // Expected output: 8
 
+const value1 = myNumbers.lastIndexOf(givenValue1)
+const value2 = myNumbers.lastIndexOf(givenValue2)
+
+console.log(`The last index of ${givenValue1} is ${value1}`)
+console.log(`The last index of ${givenValue2} is ${value2}`)
+
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
 // Pseudo code:
+// create 2 variables to call the modified variables
+// sort sanDiegoSummerTemperatures and sanDiegoWinterTemperatures
+// reverse sanDiegoSummerTemperatures and sanDiegoWinterTemperatures
+// string interpolate the console.log
 
 const sanDiegoSummerTemperatures = [79, 80, 72, 73, 82, 77, 76]
 // Expected output: [82, 80, 79, 77, 76, 73, 72]
 
 const sanDiegoWinterTemperatures = [59, 68, 62, 59, 66, 67, 66]
 // Expected output: [68, 67, 66, 66, 62, 59, 59]
+
+const tempLog = (array) => array.sort().reverse()
```

Comment:
Is there something we could do within the sort method to not have to use the reverse method?